Source/UnitTests/Common/BitUtilsTest.cpp

```diff
@@ -71,21 +71,21 @@ TEST(BitUtils, IsValidLowMask)
 
   EXPECT_TRUE(Common::IsValidLowMask((u8)~0b0));
   EXPECT_FALSE(Common::IsValidLowMask((u8)(~0b0 - 1)));
-  EXPECT_FALSE(Common::IsValidLowMask((u8)~(0b10000)));
+  EXPECT_FALSE(Common::IsValidLowMask((u8) ~(0b10000)));
```

Comment:
Maybe it believes it could be mistaken for a binary operation on first glance due to the parentheses on both sides.